src/cgame/default/cg_temp_entity.c

```diff
@@ -595,92 +628,84 @@ static void Cg_RailEffect(const vec3_t start, const vec3_t end, const vec3_t dir
 
 	cgi.AddSustainedLight(&s);
 
-	// draw the core with a beam
+	// Rail core
 
 	if (!(p = Cg_AllocParticle(PARTICLE_BEAM, cg_particles_beam))) {
 		return;
 	}
 
-	// white cores for some colors, shifted for others
-	// FIXME: what to do here for 24-bit colors?
-	ColorToVec4(ColorFromRGB(235, 235, 235), p->color_start);
```

Comment:
LOL yeah :D
There were quite a few outdated comments in the effect code; should I remove them and try to standardize effects?
Almost all the code still uses color constants and palette colors, should they be replaced with `vec4`s etc in favor of palette colors?
The code changes how the same thing is handled in different places, for example, gravity on some particles uses `VectorSet(p->accel, 0.0, 0.0, -PARTICLE_GRAVITY);` while some others use `p->accel[2] = -PARTICLE_GRAVITY`